apps/transport/lib/jobs/gtfs_to_netex_converter_job.ex

```diff
@@ -15,13 +15,20 @@ defmodule Transport.Jobs.SingleGtfsToNetexConverterJob do
   @moduledoc """
   Conversion Job of a GTFS to a NeTEx, saving the resulting file in S3
   """
-  use Oban.Worker, max_attempts: 3, queue: :heavy
+  use Oban.Worker, max_attempts: 3, queue: :heavy, unique: [period: :infinity]
```

Comment:
https://github.com/sorentwo/oban#unique-jobs

> The number of seconds until a job is no longer considered duplicate. You should always specify a period, otherwise Oban will default to 60 seconds. :infinity can be used to indicate the job be considered a duplicate as long as jobs are retained.